teamcity: failed tests on master: testrace/TestAmbiguousCommit/ambiguousSuccess=true, testrace/TestAmbiguousCommit

The following tests appear to have failed:

[#339271](https://teamcity.cockroachdb.com/viewLog.html?buildId=339271):

```
--- FAIL: testrace/TestAmbiguousCommit/ambiguousSuccess=true (0.000s)
Test ended in panic.

------- Stdout: -------
I170831 01:05:07.890723 50823 server/config.go:514  [n?] 1 storage engine initialized
I170831 01:05:07.890854 50823 server/config.go:516  [n?] RocksDB cache size: 512 MiB
I170831 01:05:07.890899 50823 server/config.go:516  [n?] store 0: in-memory, size 100 MiB
I170831 01:05:07.933506 50823 server/node.go:344  [n?] **** cluster 1e24445a-0954-4e1f-ad94-5bb9cca844fb has been created
I170831 01:05:07.933658 50823 server/server.go:824  [n?] **** add additional nodes by specifying --join=127.0.0.1:42159
I170831 01:05:07.985964 50823 server/node.go:461  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=3.2 KiB), ranges=1, leases=1, writes=0.00, bytesPerReplica={p10=3266.00 p25=3266.00 p50=3266.00 p75=3266.00 p90=3266.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00}
I170831 01:05:07.986321 50823 server/node.go:326  [n1] node ID 1 initialized
I170831 01:05:07.986693 50823 gossip/gossip.go:311  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:42159" > attrs:<> locality:<> ServerVersion:<major_val:1 minor_val:0 patch:0 unstable:3 > 
I170831 01:05:07.987514 50823 storage/stores.go:303  [n1] read 0 node addresses from persistent storage
I170831 01:05:07.987976 50823 server/node.go:606  [n1] connecting to gossip network to verify cluster ID...
I170831 01:05:07.988128 50823 server/node.go:631  [n1] node connected via gossip and verified as part of cluster "1e24445a-0954-4e1f-ad94-5bb9cca844fb"
I170831 01:05:07.988570 50823 server/node.go:403  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I170831 01:05:08.029390 50823 sql/executor.go:402  [n1] creating distSQLPlanner with address {tcp 127.0.0.1:42159}
I170831 01:05:08.050213 50945 storage/replica_command.go:2710  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I170831 01:05:08.135435 50945 storage/replica_command.go:2710  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
E170831 01:05:08.137261 51122 storage/queue.go:656  [replicate,n1,s1,r1/1:/{Min-System/}] range requires a replication change, but lacks a quorum of live replicas (0/1)
I170831 01:05:08.160904 50823 server/server.go:916  [n1] starting https server at 127.0.0.1:40547
I170831 01:05:08.161044 50823 server/server.go:917  [n1] starting grpc/postgres server at 127.0.0.1:42159
I170831 01:05:08.161087 50823 server/server.go:918  [n1] advertising CockroachDB node at 127.0.0.1:42159
W170831 01:05:08.161373 50823 sql/jobs/registry.go:156  [n1] unable to get node liveness: node not in the liveness table
I170831 01:05:08.372306 50945 storage/replica_command.go:2710  [split,n1,s1,r3/1:/{System/NodeL���-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I170831 01:05:08.435337 50823 sql/event_log.go:102  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN "uniqueID" SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I170831 01:05:08.499899 50945 storage/replica_command.go:2710  [split,n1,s1,r4/1:/{System/NodeL���-Max}] initiating a split of this range at key /System/tsd [r5]
I170831 01:05:08.500656 50823 sql/lease.go:341  [n1] publish: descID=12 (eventlog) version=2 mtime=2017-08-31 01:05:08.494591465 +0000 UTC
I170831 01:05:08.590455 50945 storage/replica_command.go:2710  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I170831 01:05:08.670010 50945 storage/replica_command.go:2710  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
E170831 01:05:08.712471 51032 storage/replica_proposal.go:538  [n1,s1,r6/1:/{System/tse-Max}] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
I170831 01:05:08.727761 50945 storage/replica_command.go:2710  [split,n1,s1,r7/1:/{Table/System���-Max}] initiating a split of this range at key /Table/11 [r8]
I170831 01:05:08.753643 50823 sql/event_log.go:102  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:node}
I170831 01:05:08.851909 50945 storage/replica_command.go:2710  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I170831 01:05:08.923048 50945 storage/replica_command.go:2710  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I170831 01:05:08.984203 50823 sql/event_log.go:102  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:1.0-3 User:node}
I170831 01:05:08.991966 50823 server/server.go:1060  [n1] done ensuring all necessary migrations have run
I170831 01:05:08.992098 50823 server/server.go:1062  [n1] serving sql connections
I170831 01:05:09.034196 50945 storage/replica_command.go:2710  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I170831 01:05:09.106245 51464 sql/event_log.go:102  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:42159} Attrs: Locality: ServerVersion:1.0-3} ClusterID:1e24445a-0954-4e1f-ad94-5bb9cca844fb StartedAt:1504141507988192018 LastUp:1504141507988192018}
I170831 01:05:09.162839 50945 storage/replica_command.go:2710  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I170831 01:05:09.320987 50945 storage/replica_command.go:2710  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I170831 01:05:09.436848 50945 storage/replica_command.go:2710  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I170831 01:05:09.520358 50945 storage/replica_command.go:2710  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I170831 01:05:09.610731 50945 storage/replica_command.go:2710  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
W170831 01:05:09.812012 50823 gossip/gossip.go:1225  [n?] no incoming or outgoing connections
I170831 01:05:09.844234 50823 server/config.go:514  [n?] 1 storage engine initialized
I170831 01:05:09.844339 50823 server/config.go:516  [n?] RocksDB cache size: 512 MiB
I170831 01:05:09.844377 50823 server/config.go:516  [n?] store 0: in-memory, size 100 MiB
I170831 01:05:09.844556 50823 server/server.go:826  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I170831 01:05:09.935877 51489 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:42159
I170831 01:05:09.955311 51770 gossip/server.go:232  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:52389}
I170831 01:05:09.982512 50823 storage/stores.go:303  [n?] read 0 node addresses from persistent storage
I170831 01:05:09.982854 50823 storage/stores.go:322  [n?] wrote 1 node addresses to persistent storage
I170831 01:05:09.982918 50823 server/node.go:606  [n?] connecting to gossip network to verify cluster ID...
I170831 01:05:09.983102 50823 server/node.go:631  [n?] node connected via gossip and verified as part of cluster "1e24445a-0954-4e1f-ad94-5bb9cca844fb"
I170831 01:05:09.992625 50823 kv/dist_sender.go:350  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170831 01:05:09.997877 50823 server/node.go:319  [n?] new node allocated ID 2
I170831 01:05:09.998208 50823 gossip/gossip.go:311  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:52389" > attrs:<> locality:<> ServerVersion:<major_val:1 minor_val:0 patch:0 unstable:3 > 
I170831 01:05:09.998756 50823 server/node.go:389  [n2] node=2: asynchronously bootstrapping engine(s) [<no-attributes>=<in-mem>]
I170831 01:05:09.999176 50823 server/node.go:403  [n2] node=2: started with [] engine(s) and attributes []
I170831 01:05:10.002601 50823 sql/executor.go:402  [n2] creating distSQLPlanner with address {tcp 127.0.0.1:52389}
I170831 01:05:10.015325 51691 storage/stores.go:322  [n1] wrote 1 node addresses to persistent storage
I170831 01:05:10.044494 50823 server/server.go:916  [n2] starting https server at 127.0.0.1:36784
I170831 01:05:10.044631 50823 server/server.go:917  [n2] starting grpc/postgres server at 127.0.0.1:52389
I170831 01:05:10.044684 50823 server/server.go:918  [n2] advertising CockroachDB node at 127.0.0.1:52389
W170831 01:05:10.045956 50823 sql/jobs/registry.go:156  [n2] unable to get node liveness: node not in the liveness table
I170831 01:05:10.069681 51807 server/node.go:587  [n2] bootstrapped store [n2,s2]
I170831 01:05:10.072985 50823 server/server.go:1060  [n2] done ensuring all necessary migrations have run
I170831 01:05:10.073109 50823 server/server.go:1062  [n2] serving sql connections
W170831 01:05:10.280754 50823 gossip/gossip.go:1225  [n?] no incoming or outgoing connections
I170831 01:05:10.291492 50823 server/config.go:514  [n?] 1 storage engine initialized
I170831 01:05:10.291615 50823 server/config.go:516  [n?] RocksDB cache size: 512 MiB
I170831 01:05:10.291675 50823 server/config.go:516  [n?] store 0: in-memory, size 100 MiB
I170831 01:05:10.291869 50823 server/server.go:826  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I170831 01:05:10.293965 51695 sql/event_log.go:102  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:52389} Attrs: Locality: ServerVersion:1.0-3} ClusterID:1e24445a-0954-4e1f-ad94-5bb9cca844fb StartedAt:1504141509998933719 LastUp:1504141509998933719}
I170831 01:05:10.347304 51787 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:42159
I170831 01:05:10.351693 51992 gossip/server.go:232  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:59331}
I170831 01:05:10.379758 50823 storage/stores.go:303  [n?] read 0 node addresses from persistent storage
I170831 01:05:10.380009 50823 storage/stores.go:322  [n?] wrote 2 node addresses to persistent storage
I170831 01:05:10.380060 50823 server/node.go:606  [n?] connecting to gossip network to verify cluster ID...
I170831 01:05:10.380161 50823 server/node.go:631  [n?] node connected via gossip and verified as part of cluster "1e24445a-0954-4e1f-ad94-5bb9cca844fb"
I170831 01:05:10.387763 50823 kv/dist_sender.go:350  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170831 01:05:10.392826 50823 server/node.go:319  [n?] new node allocated ID 3
I170831 01:05:10.396770 50823 gossip/gossip.go:311  [n3] NodeDescriptor set to node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:59331" > attrs:<> locality:<> ServerVersion:<major_val:1 minor_val:0 patch:0 unstable:3 > 
I170831 01:05:10.397294 50823 server/node.go:389  [n3] node=3: asynchronously bootstrapping engine(s) [<no-attributes>=<in-mem>]
I170831 01:05:10.397715 50823 server/node.go:403  [n3] node=3: started with [] engine(s) and attributes []
I170831 01:05:10.400063 50823 sql/executor.go:402  [n3] creating distSQLPlanner with address {tcp 127.0.0.1:59331}
I170831 01:05:10.425501 51963 storage/stores.go:322  [n1] wrote 2 node addresses to persistent storage
I170831 01:05:10.427933 52071 storage/stores.go:322  [n2] wrote 2 node addresses to persistent storage
I170831 01:05:10.452401 50823 server/server.go:916  [n3] starting https server at 127.0.0.1:47106
I170831 01:05:10.452502 50823 server/server.go:917  [n3] starting grpc/postgres server at 127.0.0.1:59331
I170831 01:05:10.452553 50823 server/server.go:918  [n3] advertising CockroachDB node at 127.0.0.1:59331
W170831 01:05:10.453858 50823 sql/jobs/registry.go:156  [n3] unable to get node liveness: node not in the liveness table
I170831 01:05:10.487072 50823 server/server.go:1060  [n3] done ensuring all necessary migrations have run
I170831 01:05:10.487192 50823 server/server.go:1062  [n3] serving sql connections
I170831 01:05:10.546458 52083 server/node.go:587  [n3] bootstrapped store [n3,s3]
I170831 01:05:10.594592 52038 sql/event_log.go:102  [n3] Event: "node_join", target: 3, info: {Descriptor:{NodeID:3 Address:{NetworkField:tcp AddressField:127.0.0.1:59331} Attrs: Locality: ServerVersion:1.0-3} ClusterID:1e24445a-0954-4e1f-ad94-5bb9cca844fb StartedAt:1504141510397491257 LastUp:1504141510397491257}
I170831 01:05:10.598485 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r7/1:/Table/{SystemCon���-11}] generated preemptive snapshot 9cd6d2bc at index 28
I170831 01:05:10.712510 51182 storage/store.go:3468  [replicate,n1,s1,r7/1:/Table/{SystemCon���-11}] streamed snapshot to (n2,s2):?: kv pairs: 44, log entries: 18, rate-limit: 8.0 MiB/sec, 3ms
I170831 01:05:10.713879 52155 storage/replica_raftstorage.go:718  [n2,s2,r7/?:{-}] applying preemptive snapshot at index 28 (id=9cd6d2bc, encoded size=20741, 1 rocksdb batches, 18 log entries)
I170831 01:05:10.715774 52155 storage/replica_raftstorage.go:726  [n2,s2,r7/?:/Table/{SystemCon���-11}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170831 01:05:10.722096 51182 storage/replica_command.go:3647  [replicate,n1,s1,r7/1:/Table/{SystemCon���-11}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r7:/Table/{SystemConfigSpan/Start-11} [(n1,s1):1, next=2]
I170831 01:05:10.741184 52232 storage/replica.go:3026  [n1,s1,r7/1:/Table/{SystemCon���-11}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170831 01:05:10.749614 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r15/1:/Table/1{8-9}] generated preemptive snapshot 830d53b8 at index 17
I170831 01:05:10.757722 52290 storage/raft_transport.go:455  [n2] raft transport stream to node 1 established
I170831 01:05:10.824032 51182 storage/store.go:3468  [replicate,n1,s1,r15/1:/Table/1{8-9}] streamed snapshot to (n3,s3):?: kv pairs: 11, log entries: 7, rate-limit: 8.0 MiB/sec, 5ms
I170831 01:05:10.825618 52288 storage/replica_raftstorage.go:718  [n3,s3,r15/?:{-}] applying preemptive snapshot at index 17 (id=830d53b8, encoded size=4688, 1 rocksdb batches, 7 log entries)
I170831 01:05:10.826969 52288 storage/replica_raftstorage.go:726  [n3,s3,r15/?:/Table/1{8-9}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:10.831313 51182 storage/replica_command.go:3647  [replicate,n1,s1,r15/1:/Table/1{8-9}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r15:/Table/1{8-9} [(n1,s1):1, next=2]
I170831 01:05:10.865326 52295 storage/replica.go:3026  [n1,s1,r15/1:/Table/1{8-9}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170831 01:05:10.872081 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r11/1:/Table/1{4-5}] generated preemptive snapshot 14148d80 at index 19
I170831 01:05:10.904115 51182 storage/store.go:3468  [replicate,n1,s1,r11/1:/Table/1{4-5}] streamed snapshot to (n2,s2):?: kv pairs: 10, log entries: 9, rate-limit: 8.0 MiB/sec, 31ms
I170831 01:05:10.905634 52235 storage/replica_raftstorage.go:718  [n2,s2,r11/?:{-}] applying preemptive snapshot at index 19 (id=14148d80, encoded size=5799, 1 rocksdb batches, 9 log entries)
I170831 01:05:10.907073 52235 storage/replica_raftstorage.go:726  [n2,s2,r11/?:/Table/1{4-5}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:10.909679 52340 storage/raft_transport.go:455  [n3] raft transport stream to node 1 established
I170831 01:05:10.910153 51182 storage/replica_command.go:3647  [replicate,n1,s1,r11/1:/Table/1{4-5}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r11:/Table/1{4-5} [(n1,s1):1, next=2]
I170831 01:05:10.943826 52353 storage/replica.go:3026  [n1,s1,r11/1:/Table/1{4-5}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170831 01:05:10.949368 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r6/1:/{System/tse-Table/System���}] generated preemptive snapshot a87e5dc3 at index 17
I170831 01:05:10.956923 51182 storage/store.go:3468  [replicate,n1,s1,r6/1:/{System/tse-Table/System���}] streamed snapshot to (n3,s3):?: kv pairs: 9, log entries: 7, rate-limit: 8.0 MiB/sec, 3ms
I170831 01:05:10.959500 52301 storage/replica_raftstorage.go:718  [n3,s3,r6/?:{-}] applying preemptive snapshot at index 17 (id=a87e5dc3, encoded size=6137, 1 rocksdb batches, 7 log entries)
I170831 01:05:10.960829 52301 storage/replica_raftstorage.go:726  [n3,s3,r6/?:/{System/tse-Table/System���}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:10.975763 51182 storage/replica_command.go:3647  [replicate,n1,s1,r6/1:/{System/tse-Table/System���}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r6:/{System/tse-Table/SystemConfigSpan/Start} [(n1,s1):1, next=2]
I170831 01:05:11.004527 52388 storage/replica.go:3026  [n1,s1,r6/1:/{System/tse-Table/System���}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170831 01:05:11.026591 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r3/1:/System/NodeLiveness{-Max}] generated preemptive snapshot dce1c62c at index 27
I170831 01:05:11.034050 52373 storage/replica_raftstorage.go:718  [n3,s3,r3/?:{-}] applying preemptive snapshot at index 27 (id=dce1c62c, encoded size=29214, 1 rocksdb batches, 17 log entries)
I170831 01:05:11.035858 52373 storage/replica_raftstorage.go:726  [n3,s3,r3/?:/System/NodeLiveness{-Max}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170831 01:05:11.042621 51182 storage/store.go:3468  [replicate,n1,s1,r3/1:/System/NodeLiveness{-Max}] streamed snapshot to (n3,s3):?: kv pairs: 13, log entries: 17, rate-limit: 8.0 MiB/sec, 15ms
I170831 01:05:11.045104 51182 storage/replica_command.go:3647  [replicate,n1,s1,r3/1:/System/NodeLiveness{-Max}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r3:/System/NodeLiveness{-Max} [(n1,s1):1, next=2]
I170831 01:05:11.123562 52237 storage/replica.go:3026  [n1,s1,r3/1:/System/NodeLiveness{-Max}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170831 01:05:11.128039 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r4/1:/System/{NodeLive���-tsd}] generated preemptive snapshot cdf0461f at index 38
I170831 01:05:11.143177 51182 storage/store.go:3468  [replicate,n1,s1,r4/1:/System/{NodeLive���-tsd}] streamed snapshot to (n2,s2):?: kv pairs: 34, log entries: 2, rate-limit: 8.0 MiB/sec, 4ms
I170831 01:05:11.145022 52334 storage/replica_raftstorage.go:718  [n2,s2,r4/?:{-}] applying preemptive snapshot at index 38 (id=cdf0461f, encoded size=65688, 1 rocksdb batches, 2 log entries)
I170831 01:05:11.151563 52334 storage/replica_raftstorage.go:726  [n2,s2,r4/?:/System/{NodeLive���-tsd}] applied preemptive snapshot in 6ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:11.162185 51182 storage/replica_command.go:3647  [replicate,n1,s1,r4/1:/System/{NodeLive���-tsd}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r4:/System/{NodeLivenessMax-tsd} [(n1,s1):1, next=2]
I170831 01:05:11.186113 52356 storage/replica.go:3026  [n1,s1,r4/1:/System/{NodeLive���-tsd}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170831 01:05:11.190342 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r13/1:/Table/1{6-7}] generated preemptive snapshot f541530b at index 17
I170831 01:05:11.193430 51182 storage/store.go:3468  [replicate,n1,s1,r13/1:/Table/1{6-7}] streamed snapshot to (n2,s2):?: kv pairs: 11, log entries: 7, rate-limit: 8.0 MiB/sec, 2ms
I170831 01:05:11.195005 52451 storage/replica_raftstorage.go:718  [n2,s2,r13/?:{-}] applying preemptive snapshot at index 17 (id=f541530b, encoded size=4684, 1 rocksdb batches, 7 log entries)
I170831 01:05:11.196236 52451 storage/replica_raftstorage.go:726  [n2,s2,r13/?:/Table/1{6-7}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:11.202620 51182 storage/replica_command.go:3647  [replicate,n1,s1,r13/1:/Table/1{6-7}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r13:/Table/1{6-7} [(n1,s1):1, next=2]
I170831 01:05:11.232778 52381 storage/replica.go:3026  [n1,s1,r13/1:/Table/1{6-7}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170831 01:05:11.238486 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r1/1:/{Min-System/}] generated preemptive snapshot 9f09132a at index 70
I170831 01:05:11.245364 51182 storage/store.go:3468  [replicate,n1,s1,r1/1:/{Min-System/}] streamed snapshot to (n3,s3):?: kv pairs: 51, log entries: 60, rate-limit: 8.0 MiB/sec, 5ms
I170831 01:05:11.247611 52367 storage/replica_raftstorage.go:718  [n3,s3,r1/?:{-}] applying preemptive snapshot at index 70 (id=9f09132a, encoded size=34575, 1 rocksdb batches, 60 log entries)
I170831 01:05:11.250647 52367 storage/replica_raftstorage.go:726  [n3,s3,r1/?:/{Min-System/}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170831 01:05:11.259437 51182 storage/replica_command.go:3647  [replicate,n1,s1,r1/1:/{Min-System/}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r1:/{Min-System/} [(n1,s1):1, next=2]
I170831 01:05:11.292718 52433 storage/replica.go:3026  [n1,s1,r1/1:/{Min-System/}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170831 01:05:11.320143 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r12/1:/Table/1{5-6}] generated preemptive snapshot 83e0f9a4 at index 18
I170831 01:05:11.324206 51182 storage/store.go:3468  [replicate,n1,s1,r12/1:/Table/1{5-6}] streamed snapshot to (n3,s3):?: kv pairs: 11, log entries: 8, rate-limit: 8.0 MiB/sec, 2ms
I170831 01:05:11.325472 52461 storage/replica_raftstorage.go:718  [n3,s3,r12/?:{-}] applying preemptive snapshot at index 18 (id=83e0f9a4, encoded size=5079, 1 rocksdb batches, 8 log entries)
I170831 01:05:11.326971 52461 storage/replica_raftstorage.go:726  [n3,s3,r12/?:/Table/1{5-6}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:11.330138 51182 storage/replica_command.go:3647  [replicate,n1,s1,r12/1:/Table/1{5-6}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r12:/Table/1{5-6} [(n1,s1):1, next=2]
I170831 01:05:11.363964 52489 storage/replica.go:3026  [n1,s1,r12/1:/Table/1{5-6}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170831 01:05:11.373938 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r14/1:/Table/1{7-8}] generated preemptive snapshot b43cd03f at index 20
I170831 01:05:11.377836 51182 storage/store.go:3468  [replicate,n1,s1,r14/1:/Table/1{7-8}] streamed snapshot to (n2,s2):?: kv pairs: 11, log entries: 10, rate-limit: 8.0 MiB/sec, 2ms
I170831 01:05:11.379244 52506 storage/replica_raftstorage.go:718  [n2,s2,r14/?:{-}] applying preemptive snapshot at index 20 (id=b43cd03f, encoded size=6361, 1 rocksdb batches, 10 log entries)
I170831 01:05:11.380615 52506 storage/replica_raftstorage.go:726  [n2,s2,r14/?:/Table/1{7-8}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:11.387531 51182 storage/replica_command.go:3647  [replicate,n1,s1,r14/1:/Table/1{7-8}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r14:/Table/1{7-8} [(n1,s1):1, next=2]
I170831 01:05:11.450974 52507 storage/replica.go:3026  [n1,s1,r14/1:/Table/1{7-8}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170831 01:05:11.461016 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r5/1:/System/ts{d-e}] generated preemptive snapshot 17e813a0 at index 28
I170831 01:05:11.532684 52532 storage/replica_raftstorage.go:718  [n2,s2,r5/?:{-}] applying preemptive snapshot at index 28 (id=17e813a0, encoded size=121811, 1 rocksdb batches, 2 log entries)
I170831 01:05:11.533379 51182 storage/store.go:3468  [replicate,n1,s1,r5/1:/System/ts{d-e}] streamed snapshot to (n2,s2):?: kv pairs: 922, log entries: 2, rate-limit: 8.0 MiB/sec, 68ms
I170831 01:05:11.535137 52532 storage/replica_raftstorage.go:726  [n2,s2,r5/?:/System/ts{d-e}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=1ms]
I170831 01:05:11.542836 51182 storage/replica_command.go:3647  [replicate,n1,s1,r5/1:/System/ts{d-e}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r5:/System/ts{d-e} [(n1,s1):1, next=2]
I170831 01:05:11.606294 52551 storage/replica.go:3026  [n1,s1,r5/1:/System/ts{d-e}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170831 01:05:11.643064 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r16/1:/{Table/19-Max}] generated preemptive snapshot abb2b038 at index 11
I170831 01:05:11.651260 51182 storage/store.go:3468  [replicate,n1,s1,r16/1:/{Table/19-Max}] streamed snapshot to (n3,s3):?: kv pairs: 9, log entries: 1, rate-limit: 8.0 MiB/sec, 7ms
I170831 01:05:11.652996 52596 storage/replica_raftstorage.go:718  [n3,s3,r16/?:{-}] applying preemptive snapshot at index 11 (id=abb2b038, encoded size=543, 1 rocksdb batches, 1 log entries)
I170831 01:05:11.654151 52596 storage/replica_raftstorage.go:726  [n3,s3,r16/?:/{Table/19-Max}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:11.661625 51182 storage/replica_command.go:3647  [replicate,n1,s1,r16/1:/{Table/19-Max}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r16:/{Table/19-Max} [(n1,s1):1, next=2]
I170831 01:05:11.715860 52610 storage/replica.go:3026  [n1,s1,r16/1:/{Table/19-Max}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170831 01:05:11.730151 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r8/1:/Table/1{1-2}] generated preemptive snapshot 1d1ff027 at index 18
I170831 01:05:11.734198 51182 storage/store.go:3468  [replicate,n1,s1,r8/1:/Table/1{1-2}] streamed snapshot to (n3,s3):?: kv pairs: 11, log entries: 8, rate-limit: 8.0 MiB/sec, 2ms
I170831 01:05:11.735866 52602 storage/replica_raftstorage.go:718  [n3,s3,r8/?:{-}] applying preemptive snapshot at index 18 (id=1d1ff027, encoded size=6354, 1 rocksdb batches, 8 log entries)
I170831 01:05:11.737131 52602 storage/replica_raftstorage.go:726  [n3,s3,r8/?:/Table/1{1-2}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:11.748766 51182 storage/replica_command.go:3647  [replicate,n1,s1,r8/1:/Table/1{1-2}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r8:/Table/1{1-2} [(n1,s1):1, next=2]
I170831 01:05:11.790378 52394 storage/replica.go:3026  [n1,s1,r8/1:/Table/1{1-2}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170831 01:05:11.830915 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r10/1:/Table/1{3-4}] generated preemptive snapshot 5f68387b at index 59
I170831 01:05:11.850389 52537 storage/replica_raftstorage.go:718  [n2,s2,r10/?:{-}] applying preemptive snapshot at index 59 (id=5f68387b, encoded size=75248, 1 rocksdb batches, 49 log entries)
I170831 01:05:11.853242 52537 storage/replica_raftstorage.go:726  [n2,s2,r10/?:/Table/1{3-4}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170831 01:05:11.858186 51182 storage/store.go:3468  [replicate,n1,s1,r10/1:/Table/1{3-4}] streamed snapshot to (n2,s2):?: kv pairs: 165, log entries: 49, rate-limit: 8.0 MiB/sec, 23ms
I170831 01:05:11.864757 51182 storage/replica_command.go:3647  [replicate,n1,s1,r10/1:/Table/1{3-4}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r10:/Table/1{3-4} [(n1,s1):1, next=2]
I170831 01:05:11.894521 52630 storage/replica.go:3026  [n1,s1,r10/1:/Table/1{3-4}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170831 01:05:11.904923 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r9/1:/Table/1{2-3}] generated preemptive snapshot ee74d144 at index 27
I170831 01:05:11.908251 51182 storage/store.go:3468  [replicate,n1,s1,r9/1:/Table/1{2-3}] streamed snapshot to (n2,s2):?: kv pairs: 41, log entries: 17, rate-limit: 8.0 MiB/sec, 3ms
I170831 01:05:11.917448 52633 storage/replica_raftstorage.go:718  [n2,s2,r9/?:{-}] applying preemptive snapshot at index 27 (id=ee74d144, encoded size=19694, 1 rocksdb batches, 17 log entries)
I170831 01:05:11.919293 52633 storage/replica_raftstorage.go:726  [n2,s2,r9/?:/Table/1{2-3}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170831 01:05:11.923189 51182 storage/replica_command.go:3647  [replicate,n1,s1,r9/1:/Table/1{2-3}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r9:/Table/1{2-3} [(n1,s1):1, next=2]
I170831 01:05:11.964741 52674 storage/replica.go:3026  [n1,s1,r9/1:/Table/1{2-3}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170831 01:05:11.973359 51182 storage/replica_raftstorage.go:509  [replicate,n1,s1,r2/1:/System/{-NodeLive���}] generated preemptive snapshot 94f4accf at index 22
I170831 01:05:12.024426 51182 storage/store.go:3468  [replicate,n1,s1,r2/1:/System/{-NodeLive���}] streamed snapshot to (n3,s3):?: kv pairs: 10, log entries: 12, rate-limit: 8.0 MiB/sec, 47ms
I170831 01:05:12.026660 52522 storage/replica_raftstorage.go:718  [n3,s3,r2/?:{-}] applying preemptive snapshot at index 22 (id=94f4accf, encoded size=54982, 1 rocksdb batches, 12 log entries)
I170831 01:05:12.028715 52522 storage/replica_raftstorage.go:726  [n3,s3,r2/?:/System/{-NodeLive���}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170831 01:05:12.035445 51182 storage/replica_command.go:3647  [replicate,n1,s1,r2/1:/System/{-NodeLive���}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r2:/System/{-NodeLiveness} [(n1,s1):1, next=2]
I170831 01:05:12.067801 52646 storage/replica.go:3026  [n1,s1,r2/1:/System/{-NodeLive���}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170831 01:05:12.084381 51182 storage/queue.go:723  [n1,replicate] purgatory is now empty
I170831 01:05:12.085721 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r5/1:/System/ts{d-e}] generated preemptive snapshot 26bf645d at index 31
I170831 01:05:12.111441 51122 storage/store.go:3468  [replicate,n1,s1,r5/1:/System/ts{d-e}] streamed snapshot to (n3,s3):?: kv pairs: 923, log entries: 5, rate-limit: 8.0 MiB/sec, 25ms
I170831 01:05:12.113769 52694 storage/replica_raftstorage.go:718  [n3,s3,r5/?:{-}] applying preemptive snapshot at index 31 (id=26bf645d, encoded size=123911, 1 rocksdb batches, 5 log entries)
I170831 01:05:12.115962 52694 storage/replica_raftstorage.go:726  [n3,s3,r5/?:/System/ts{d-e}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170831 01:05:12.120329 51122 storage/replica_command.go:3647  [replicate,n1,s1,r5/1:/System/ts{d-e}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r5:/System/ts{d-e} [(n1,s1):1, (n2,s2):2, next=3]
I170831 01:05:12.167491 52652 storage/replica.go:3026  [n1,s1,r5/1:/System/ts{d-e}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I170831 01:05:12.184822 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r8/1:/Table/1{1-2}] generated preemptive snapshot a76a9032 at index 21
I170831 01:05:12.190557 51122 storage/store.go:3468  [replicate,n1,s1,r8/1:/Table/1{1-2}] streamed snapshot to (n2,s2):?: kv pairs: 12, log entries: 11, rate-limit: 8.0 MiB/sec, 5ms
I170831 01:05:12.193042 52574 storage/replica_raftstorage.go:718  [n2,s2,r8/?:{-}] applying preemptive snapshot at index 21 (id=a76a9032, encoded size=8385, 1 rocksdb batches, 11 log entries)
I170831 01:05:12.205849 52574 storage/replica_raftstorage.go:726  [n2,s2,r8/?:/Table/1{1-2}] applied preemptive snapshot in 13ms [clear=0ms batch=0ms entries=10ms commit=2ms]
I170831 01:05:12.215422 51122 storage/replica_command.go:3647  [replicate,n1,s1,r8/1:/Table/1{1-2}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r8:/Table/1{1-2} [(n1,s1):1, (n3,s3):2, next=3]
I170831 01:05:12.286317 52724 storage/replica.go:3026  [n1,s1,r8/1:/Table/1{1-2}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170831 01:05:12.298923 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r14/1:/Table/1{7-8}] generated preemptive snapshot 5c846ad9 at index 23
I170831 01:05:12.304348 51122 storage/store.go:3468  [replicate,n1,s1,r14/1:/Table/1{7-8}] streamed snapshot to (n3,s3):?: kv pairs: 12, log entries: 13, rate-limit: 8.0 MiB/sec, 4ms
I170831 01:05:12.308884 52655 storage/replica_raftstorage.go:718  [n3,s3,r14/?:{-}] applying preemptive snapshot at index 23 (id=5c846ad9, encoded size=8392, 1 rocksdb batches, 13 log entries)
I170831 01:05:12.310329 52655 storage/replica_raftstorage.go:726  [n3,s3,r14/?:/Table/1{7-8}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:12.316048 51122 storage/replica_command.go:3647  [replicate,n1,s1,r14/1:/Table/1{7-8}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r14:/Table/1{7-8} [(n1,s1):1, (n2,s2):2, next=3]
I170831 01:05:12.374286 52755 storage/replica.go:3026  [n1,s1,r14/1:/Table/1{7-8}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I170831 01:05:12.382963 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r2/1:/System/{-NodeLive���}] generated preemptive snapshot de44bf08 at index 25
I170831 01:05:12.403135 51122 storage/store.go:3468  [replicate,n1,s1,r2/1:/System/{-NodeLive���}] streamed snapshot to (n2,s2):?: kv pairs: 11, log entries: 15, rate-limit: 8.0 MiB/sec, 19ms
I170831 01:05:12.404874 52750 storage/replica_raftstorage.go:718  [n2,s2,r2/?:{-}] applying preemptive snapshot at index 25 (id=de44bf08, encoded size=57053, 1 rocksdb batches, 15 log entries)
I170831 01:05:12.408386 52750 storage/replica_raftstorage.go:726  [n2,s2,r2/?:/System/{-NodeLive���}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170831 01:05:12.412053 51122 storage/replica_command.go:3647  [replicate,n1,s1,r2/1:/System/{-NodeLive���}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r2:/System/{-NodeLiveness} [(n1,s1):1, (n3,s3):2, next=3]
I170831 01:05:12.476072 52699 storage/replica.go:3026  [n1,s1,r2/1:/System/{-NodeLive���}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170831 01:05:12.498958 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r12/1:/Table/1{5-6}] generated preemptive snapshot 4e010134 at index 21
I170831 01:05:12.538626 51122 storage/store.go:3468  [replicate,n1,s1,r12/1:/Table/1{5-6}] streamed snapshot to (n2,s2):?: kv pairs: 12, log entries: 11, rate-limit: 8.0 MiB/sec, 39ms
I170831 01:05:12.540900 52764 storage/replica_raftstorage.go:718  [n2,s2,r12/?:{-}] applying preemptive snapshot at index 21 (id=4e010134, encoded size=7110, 1 rocksdb batches, 11 log entries)
I170831 01:05:12.542597 52764 storage/replica_raftstorage.go:726  [n2,s2,r12/?:/Table/1{5-6}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:12.549246 51122 storage/replica_command.go:3647  [replicate,n1,s1,r12/1:/Table/1{5-6}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r12:/Table/1{5-6} [(n1,s1):1, (n3,s3):2, next=3]
I170831 01:05:12.661686 52768 storage/replica.go:3026  [n1,s1,r12/1:/Table/1{5-6}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170831 01:05:12.691453 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r9/1:/Table/1{2-3}] generated preemptive snapshot 7607a38b at index 30
I170831 01:05:12.705284 51122 storage/store.go:3468  [replicate,n1,s1,r9/1:/Table/1{2-3}] streamed snapshot to (n3,s3):?: kv pairs: 42, log entries: 20, rate-limit: 8.0 MiB/sec, 12ms
I170831 01:05:12.724873 52719 storage/replica_raftstorage.go:718  [n3,s3,r9/?:{-}] applying preemptive snapshot at index 30 (id=7607a38b, encoded size=21728, 1 rocksdb batches, 20 log entries)
I170831 01:05:12.737145 52719 storage/replica_raftstorage.go:726  [n3,s3,r9/?:/Table/1{2-3}] applied preemptive snapshot in 12ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170831 01:05:12.746281 51122 storage/replica_command.go:3647  [replicate,n1,s1,r9/1:/Table/1{2-3}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r9:/Table/1{2-3} [(n1,s1):1, (n2,s2):2, next=3]
I170831 01:05:12.787884 52721 storage/replica.go:3026  [n1,s1,r9/1:/Table/1{2-3}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I170831 01:05:12.799815 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r10/1:/Table/1{3-4}] generated preemptive snapshot 6c100cc0 at index 81
I170831 01:05:12.806581 51122 storage/store.go:3468  [replicate,n1,s1,r10/1:/Table/1{3-4}] streamed snapshot to (n3,s3):?: kv pairs: 217, log entries: 15, rate-limit: 8.0 MiB/sec, 5ms
I170831 01:05:12.809018 52737 storage/replica_raftstorage.go:718  [n3,s3,r10/?:{-}] applying preemptive snapshot at index 81 (id=6c100cc0, encoded size=44161, 1 rocksdb batches, 15 log entries)
I170831 01:05:12.833569 52737 storage/replica_raftstorage.go:726  [n3,s3,r10/?:/Table/1{3-4}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170831 01:05:12.849316 51122 storage/replica_command.go:3647  [replicate,n1,s1,r10/1:/Table/1{3-4}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r10:/Table/1{3-4} [(n1,s1):1, (n2,s2):2, next=3]
I170831 01:05:12.934424 52702 storage/replica.go:3026  [n1,s1,r10/1:/Table/1{3-4}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I170831 01:05:12.961301 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r3/1:/System/NodeLiveness{-Max}] generated preemptive snapshot a2c4e73e at index 32
I170831 01:05:12.972616 51122 storage/store.go:3468  [replicate,n1,s1,r3/1:/System/NodeLiveness{-Max}] streamed snapshot to (n2,s2):?: kv pairs: 15, log entries: 22, rate-limit: 8.0 MiB/sec, 11ms
I170831 01:05:12.974314 52822 storage/replica_raftstorage.go:718  [n2,s2,r3/?:{-}] applying preemptive snapshot at index 32 (id=a2c4e73e, encoded size=32550, 1 rocksdb batches, 22 log entries)
I170831 01:05:12.976220 52822 storage/replica_raftstorage.go:726  [n2,s2,r3/?:/System/NodeLiveness{-Max}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170831 01:05:12.985694 51122 storage/replica_command.go:3647  [replicate,n1,s1,r3/1:/System/NodeLiveness{-Max}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r3:/System/NodeLiveness{-Max} [(n1,s1):1, (n3,s3):2, next=3]
I170831 01:05:13.032820 52890 storage/replica.go:3026  [n1,s1,r3/1:/System/NodeLiveness{-Max}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170831 01:05:13.058008 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r15/1:/Table/1{8-9}] generated preemptive snapshot 3d4724dc at index 20
I170831 01:05:13.098778 51122 storage/store.go:3468  [replicate,n1,s1,r15/1:/Table/1{8-9}] streamed snapshot to (n2,s2):?: kv pairs: 12, log entries: 10, rate-limit: 8.0 MiB/sec, 23ms
I170831 01:05:13.120758 52931 storage/replica_raftstorage.go:718  [n2,s2,r15/?:{-}] applying preemptive snapshot at index 20 (id=3d4724dc, encoded size=6719, 1 rocksdb batches, 10 log entries)
I170831 01:05:13.124239 52931 storage/replica_raftstorage.go:726  [n2,s2,r15/?:/Table/1{8-9}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=0ms commit=2ms]
I170831 01:05:13.131434 51122 storage/replica_command.go:3647  [replicate,n1,s1,r15/1:/Table/1{8-9}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r15:/Table/1{8-9} [(n1,s1):1, (n3,s3):2, next=3]
I170831 01:05:13.174114 52778 storage/replica.go:3026  [n1,s1,r15/1:/Table/1{8-9}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170831 01:05:13.191416 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r16/1:/{Table/19-Max}] generated preemptive snapshot 48dcc264 at index 14
I170831 01:05:13.197629 52921 storage/replica_raftstorage.go:718  [n2,s2,r16/?:{-}] applying preemptive snapshot at index 14 (id=48dcc264, encoded size=2578, 1 rocksdb batches, 4 log entries)
I170831 01:05:13.198967 52921 storage/replica_raftstorage.go:726  [n2,s2,r16/?:/{Table/19-Max}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:13.199318 51122 storage/store.go:3468  [replicate,n1,s1,r16/1:/{Table/19-Max}] streamed snapshot to (n2,s2):?: kv pairs: 10, log entries: 4, rate-limit: 8.0 MiB/sec, 7ms
I170831 01:05:13.206501 51122 storage/replica_command.go:3647  [replicate,n1,s1,r16/1:/{Table/19-Max}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r16:/{Table/19-Max} [(n1,s1):1, (n3,s3):2, next=3]
I170831 01:05:13.251665 52948 storage/replica.go:3026  [n1,s1,r16/1:/{Table/19-Max}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170831 01:05:13.304051 51900 storage/replica_proposal.go:453  [n2,s2,r5/2:/System/ts{d-e}] new range lease repl=(n2,s2):2 start=1504141513.279140927,0 epo=1 pro=1504141513.279150327,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1504141516.974259640,0 pro=1504141507.974304040,0
I170831 01:05:13.310556 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r7/1:/Table/{SystemCon���-11}] generated preemptive snapshot f16db8c2 at index 31
I170831 01:05:13.360802 51122 storage/store.go:3468  [replicate,n1,s1,r7/1:/Table/{SystemCon���-11}] streamed snapshot to (n3,s3):?: kv pairs: 45, log entries: 21, rate-limit: 8.0 MiB/sec, 34ms
I170831 01:05:13.402267 52969 storage/replica_raftstorage.go:718  [n3,s3,r7/?:{-}] applying preemptive snapshot at index 31 (id=f16db8c2, encoded size=22772, 1 rocksdb batches, 21 log entries)
I170831 01:05:13.404559 52969 storage/replica_raftstorage.go:726  [n3,s3,r7/?:/Table/{SystemCon���-11}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170831 01:05:13.417322 51122 storage/replica_command.go:3647  [replicate,n1,s1,r7/1:/Table/{SystemCon���-11}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r7:/Table/{SystemConfigSpan/Start-11} [(n1,s1):1, (n2,s2):2, next=3]
I170831 01:05:13.501040 52925 storage/replica.go:3026  [n1,s1,r7/1:/Table/{SystemCon���-11}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I170831 01:05:13.545930 52780 storage/raft_transport.go:455  [n2] raft transport stream to node 3 established
I170831 01:05:13.551110 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r1/1:/{Min-System/}] generated preemptive snapshot ea765974 at index 113
I170831 01:05:13.560007 51122 storage/store.go:3468  [replicate,n1,s1,r1/1:/{Min-System/}] streamed snapshot to (n2,s2):?: kv pairs: 72, log entries: 103, rate-limit: 8.0 MiB/sec, 8ms
I170831 01:05:13.564402 52955 storage/replica_raftstorage.go:718  [n2,s2,r1/?:{-}] applying preemptive snapshot at index 113 (id=ea765974, encoded size=56811, 1 rocksdb batches, 103 log entries)
I170831 01:05:13.567899 52955 storage/replica_raftstorage.go:726  [n2,s2,r1/?:/{Min-System/}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170831 01:05:13.574947 51122 storage/replica_command.go:3647  [replicate,n1,s1,r1/1:/{Min-System/}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r1:/{Min-System/} [(n1,s1):1, (n3,s3):2, next=3]
I170831 01:05:13.633630 53031 storage/replica.go:3026  [n1,s1,r1/1:/{Min-System/}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170831 01:05:13.650610 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r6/1:/{System/tse-Table/System���}] generated preemptive snapshot ff860d9d at index 23
I170831 01:05:13.659558 51122 storage/store.go:3468  [replicate,n1,s1,r6/1:/{System/tse-Table/System���}] streamed snapshot to (n2,s2):?: kv pairs: 12, log entries: 13, rate-limit: 8.0 MiB/sec, 8ms
I170831 01:05:13.664951 53033 storage/replica_raftstorage.go:718  [n2,s2,r6/?:{-}] applying preemptive snapshot at index 23 (id=ff860d9d, encoded size=9735, 1 rocksdb batches, 13 log entries)
I170831 01:05:13.667346 53033 storage/replica_raftstorage.go:726  [n2,s2,r6/?:/{System/tse-Table/System���}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170831 01:05:13.676866 51128 storage/allocator_scorer.go:639  [replicate,n1,s1,r1/1:/{Min-System/}] s2: should-rebalance(better-fit-ranges=s3): rangeCount=16, otherRangeCount=13, mean=15.00, underfull-threshold=14
I170831 01:05:13.688877 51122 storage/replica_command.go:3647  [replicate,n1,s1,r6/1:/{System/tse-Table/System���}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r6:/{System/tse-Table/SystemConfigSpan/Start} [(n1,s1):1, (n3,s3):2, next=3]
I170831 01:05:13.740822 53044 storage/replica.go:3026  [n1,s1,r6/1:/{System/tse-Table/System���}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170831 01:05:13.752294 51122 storage/allocator_scorer.go:639  [replicate,n1,s1,r6/1:/{System/tse-Table/System���}] s2: should-rebalance(better-fit-ranges=s3): rangeCount=16, otherRangeCount=13, mean=15.00, underfull-threshold=14
I170831 01:05:13.754150 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r11/1:/Table/1{4-5}] generated preemptive snapshot 7cbc0b10 at index 24
I170831 01:05:13.756653 53026 storage/raft_transport.go:455  [n3] raft transport stream to node 2 established
I170831 01:05:13.764060 51122 storage/store.go:3468  [replicate,n1,s1,r11/1:/Table/1{4-5}] streamed snapshot to (n3,s3):?: kv pairs: 12, log entries: 14, rate-limit: 8.0 MiB/sec, 9ms
I170831 01:05:13.765451 52988 storage/replica_raftstorage.go:718  [n3,s3,r11/?:{-}] applying preemptive snapshot at index 24 (id=7cbc0b10, encoded size=8787, 1 rocksdb batches, 14 log entries)
I170831 01:05:13.766880 52988 storage/replica_raftstorage.go:726  [n3,s3,r11/?:/Table/1{4-5}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:13.774055 51122 storage/replica_command.go:3647  [replicate,n1,s1,r11/1:/Table/1{4-5}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r11:/Table/1{4-5} [(n1,s1):1, (n2,s2):2, next=3]
I170831 01:05:13.863219 53090 storage/replica.go:3026  [n1,s1,r11/1:/Table/1{4-5}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I170831 01:05:13.876861 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r13/1:/Table/1{6-7}] generated preemptive snapshot f92eee1a at index 20
I170831 01:05:13.911452 53072 storage/replica_raftstorage.go:718  [n3,s3,r13/?:{-}] applying preemptive snapshot at index 20 (id=f92eee1a, encoded size=6715, 1 rocksdb batches, 10 log entries)
I170831 01:05:13.912898 53072 storage/replica_raftstorage.go:726  [n3,s3,r13/?:/Table/1{6-7}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:13.914215 51122 storage/store.go:3468  [replicate,n1,s1,r13/1:/Table/1{6-7}] streamed snapshot to (n3,s3):?: kv pairs: 12, log entries: 10, rate-limit: 8.0 MiB/sec, 33ms
I170831 01:05:13.916240 51122 storage/replica_command.go:3647  [replicate,n1,s1,r13/1:/Table/1{6-7}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r13:/Table/1{6-7} [(n1,s1):1, (n2,s2):2, next=3]
I170831 01:05:13.963555 52990 storage/replica.go:3026  [n1,s1,r13/1:/Table/1{6-7}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I170831 01:05:13.972337 51122 storage/replica_raftstorage.go:509  [replicate,n1,s1,r4/1:/System/{NodeLive���-tsd}] generated preemptive snapshot e6bacd4e at index 41
I170831 01:05:14.006770 51122 storage/store.go:3468  [replicate,n1,s1,r4/1:/System/{NodeLive���-tsd}] streamed snapshot to (n3,s3):?: kv pairs: 35, log entries: 5, rate-limit: 8.0 MiB/sec, 34ms
I170831 01:05:14.010589 52991 storage/replica_raftstorage.go:718  [n3,s3,r4/?:{-}] applying preemptive snapshot at index 41 (id=e6bacd4e, encoded size=67931, 1 rocksdb batches, 5 log entries)
I170831 01:05:14.018526 52991 storage/replica_raftstorage.go:726  [n3,s3,r4/?:/System/{NodeLive���-tsd}] applied preemptive snapshot in 8ms [clear=6ms batch=0ms entries=0ms commit=0ms]
I170831 01:05:14.029465 51122 storage/replica_command.go:3647  [replicate,n1,s1,r4/1:/System/{NodeLive���-tsd}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r4:/System/{NodeLivenessMax-tsd} [(n1,s1):1, (n2,s2):2, next=3]
I170831 01:05:14.066698 53091 storage/replica.go:3026  [n1,s1,r4/1:/System/{NodeLive���-tsd}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I170831 01:05:14.377538 53186 sql/event_log.go:102  [client=127.0.0.1:60923,user=root,n1] Event: "create_database", target: 50, info: {DatabaseName:test Statement:CREATE DATABASE test User:root}
I170831 01:05:14.378361 50945 storage/replica_command.go:2710  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/50 [r17]
I170831 01:05:14.506509 53186 sql/event_log.go:102  [client=127.0.0.1:60923,user=root,n1] Event: "create_table", target: 51, info: {TableName:test.t Statement:CREATE TABLE test.t (k SERIAL PRIMARY KEY, v INT) User:root}
I170831 01:05:14.530897 50945 storage/replica_command.go:2710  [split,n1,s1,r17/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r18]
I170831 01:05:14.545057 52162 storage/replica_proposal.go:453  [n3,s3,r8/2:/Table/1{1-2}] new range lease repl=(n3,s3):2 start=1504141514.491558822,0 epo=1 pro=1504141514.491567622,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1504141516.974259640,0 pro=1504141507.974304040,0
W170831 01:05:14.966913 53026 storage/raft_transport.go:461  [n3] raft transport stream to node 2 failed: rpc error: code = Unavailable desc = transport is closing
I170831 01:05:14.967515 53254 util/stop/stopper.go:476  quiescing; tasks left:
1      node.Node: batch
W170831 01:05:14.967815 53306 storage/replica.go:2667  [n1,s1,r3/1:/System/NodeLiveness{-Max}] context cancellation after 0.0s of attempting command [txn: 6c5549c6], BeginTransaction [/System/NodeLiveness/3,/Min), ConditionalPut [/System/NodeLiveness/3,/Min), EndTransaction [/System/NodeLiveness/3,/Min)
I170831 01:05:14.968308 53306 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  grpc: Server.processUnaryRPC failed to write status stream error: code = Canceled desc = "context canceled"
W170831 01:05:14.968578 52340 storage/raft_transport.go:461  [n3] raft transport stream to node 1 failed: rpc error: code = Unavailable desc = transport is closing
W170831 01:05:14.969779 52290 storage/raft_transport.go:461  [n2] raft transport stream to node 1 failed: EOF
W170831 01:05:14.971267 52252 storage/raft_transport.go:461  [n1] raft transport stream to node 3 failed: rpc error: code = Unavailable desc = transport is closing
E170831 01:05:14.971668 52127 util/log/crash_reporting.go:108  [n3] a panic has occurred!
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1e266ab]

goroutine 52127 [running]:
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Recover(0xc4212705a0, 0x7f507a4c4800, 0xc42077e900)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:183 +0x100
panic(0x21fc060, 0x330e290)
	/usr/local/go/src/runtime/panic.go:489 +0x2f0
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).divideAndSendBatchToRanges.func1(0xc4238d2008, 0xc4238d2788, 0xc4238d2780, 0xc4238d1f0a, 0xc4238d26e0, 0xc4238d1ff0)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:663 +0x510
panic(0x21fc060, 0x330e290)
	/usr/local/go/src/runtime/panic.go:489 +0x2f0
github.com/cockroachdb/cockroach/pkg/sql_test.TestAmbiguousCommit.func1.2.1(0x7f507b69ec98, 0xc421e0cc00, 0xc420b9e6c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/ambiguous_commit_test.go:97 +0x17b
github.com/cockroachdb/cockroach/pkg/sql_test.(*interceptingTransport).SendNext(0xc420315960, 0x7f507b69ec98, 0xc421e0cc00, 0xc420b9e6c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/ambiguous_commit_test.go:47 +0x71
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).sendToReplicas(0xc4201bc1e0, 0x7f507b69ec98, 0xc421e0cc00, 0xc4201bc230, 0x3, 0xc420315920, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:1135 +0x6bd
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).sendRPC(0xc4201bc1e0, 0x7f507b69ec98, 0xc421e0cc00, 0x3, 0xc420315920, 0x1, 0x1, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:382 +0x353
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).sendSingleRange(0xc4201bc1e0, 0x7f507a4c4800, 0xc420cd0780, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc420e1aa80, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:446 +0x1b8
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).sendPartialBatch(0xc4201bc1e0, 0x7f507a4c4800, 0xc420cd0780, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc420e1aa80, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:930 +0x575
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).divideAndSendBatchToRanges(0xc4201bc1e0, 0x7f507a4c4800, 0xc420cd0780, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc420e1aa80, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:795 +0xd1e
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).Send(0xc4201bc1e0, 0x7f507a4c4800, 0xc420cd0780, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc420e1a9a0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:597 +0x3ee
github.com/cockroachdb/cockroach/pkg/kv.(*TxnCoordSender).Send(0xc4200a4ee0, 0x7f507a4c4800, 0xc420cd0330, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc420e1a7e0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:444 +0x282
github.com/cockroachdb/cockroach/pkg/internal/client.(*DB).send(0xc4204027e0, 0x7f507b69ec98, 0xc421e0c980, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc420e1a7e0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:545 +0x20a
github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).Send(0xc4220914a0, 0x7f507b69ec98, 0xc421e0c980, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:908 +0x578
github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).Send-fm(0x7f507b69ec98, 0xc421e0c980, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:486 +0xa4
github.com/cockroachdb/cockroach/pkg/internal/client.sendAndFill(0x7f507b69ec98, 0xc421e0c980, 0xc421d4f500, 0xc4223a1c00, 0xc4200a6690, 0xc421d4f628)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:454 +0x156
github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).Run(0xc4220914a0, 0x7f507b69ec98, 0xc421e0c980, 0xc4223a1c00, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:486 +0x112
github.com/cockroachdb/cockroach/pkg/storage.(*NodeLiveness).updateLivenessAttempt.func1(0x7f507b69ec98, 0xc421e0c980, 0xc4220914a0, 0x2, 0x22656c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/node_liveness.go:662 +0x599
github.com/cockroachdb/cockroach/pkg/internal/client.(*DB).Txn.func1(0x7f507b69ec98, 0xc421e0c980, 0xc4220914a0, 0xc424a00662, 0x14dfc8bd12b9cf77, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:507 +0x5f
github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).Exec(0xc4220914a0, 0x7f507b69ec98, 0xc421e0c980, 0xc421e00101, 0xc424a006d0, 0x28, 0xc420ec05a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:722 +0xc6
github.com/cockroachdb/cockroach/pkg/internal/client.(*DB).Txn(0xc4204027e0, 0x7f507b69ec98, 0xc421e0c980, 0xc420315720, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:508 +0x175
github.com/cockroachdb/cockroach/pkg/storage.(*NodeLiveness).updateLivenessAttempt(0xc420ec0500, 0x7f507a4c4800, 0xc420cd0090, 0xc420cd0120, 0xc420cd00f0, 0xc424683b80, 0xc420f072d0, 0xc4223bea70)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/node_liveness.go:663 +0x18d
github.com/cockroachdb/cockroach/pkg/storage.(*NodeLiveness).updateLiveness(0xc420ec0500, 0x7f507a4c4800, 0xc420cd0090, 0xc420cd0120, 0xc420cd00f0, 0xc424683b80, 0x380b120, 0x6d128b)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/node_liveness.go:613 +0x7e
github.com/cockroachdb/cockroach/pkg/storage.(*NodeLiveness).heartbeatInternal(0xc420ec0500, 0x7f507a4c4800, 0xc420cd0090, 0xc420cd00f0, 0x4000000000000000, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/node_liveness.go:438 +0x4aa
github.com/cockroachdb/cockroach/pkg/storage.(*NodeLiveness).StartHeartbeat.func1.1(0xc420ec0500, 0xc4223bee80, 0x2faf080, 0x3b9aca00, 0x4000000000000000, 0x0, 0x0, 0xc4235a2540, 0xc424683e37)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/node_liveness.go:338 +0x30e
github.com/cockroachdb/cockroach/pkg/storage.(*NodeLiveness).StartHeartbeat.func1(0x7f507a4c4800, 0xc42077e900)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/node_liveness.go:349 +0x37d
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202bba10, 0xc4212705a0, 0xc422bf6af0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:199 +0xbb

goroutine 1 [chan receive]:
testing.(*T).Run(0xc420240820, 0x23e46e6, 0x13, 0x24acb78, 0xc420521b01)
	/usr/local/go/src/testing/testing.go:698 +0x582
testing.runTests.func1(0xc420240820)
	/usr/local
```
Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/ambiguous_commit_test.go
+++ b/pkg/sql/ambiguous_commit_test.go
@@ -94,10 +94,16 @@ func TestAmbiguousCommit(t *testing.T) {
 								interceptDone := make(chan kv.BatchCall)
 								transport.SendNext(ctx, interceptDone)
 								call := <-interceptDone
-								if pErr := call.Reply.Error; pErr.Equal(translateToRPCError) {
+								// During shutdown, we may get responses that
+								// have call.Err set and all we have to do is
+								// not crash on those.
+								//
+								// For the rest, compare and perhaps inject an
+								// RPC error ourselves.
+								if call.Err == nil && call.Reply.Error.Equal(translateToRPCError) {
 									// Translate the injected error into an RPC
 									// error to simulate an ambiguous result.
-									done <- kv.BatchCall{Err: pErr.GoError()}
+									done <- kv.BatchCall{Err: call.Reply.Error.GoError()}
 								} else {
 									// Either the call succeeded or we got a non-
 									// sentinel error; let normal machinery do its

